code/Test_definitions/carrier-billing-createPayment.feature

```diff
@@ -0,0 +1,428 @@
+    And the response body property "$.paymentStatus" is "processing"
+
+
+  ##############################
+  # Error scenarios
+  ##############################
+
+  # Error 400 scenarios
+
+  @create_payment_400.01_no_request_body
+  Scenario: Missing request body
+    Given the request body is not included
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+
+  @create_payment_400.02_empty_request_body
+  Scenario: Empty object as request body
+    Given the request body is set to "{}"
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+
+  @create_payment_400.03_other_input_properties_schema_not_compliant
+  # Test other input properties in addition to amountTransaction
+  Scenario Outline: Input property values does not comply with the schema
+    Given the request body property "<input_property>" does not comply with the OAS schema at <oas_spec_schema>
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+    Examples:
+      | input_property          | oas_spec_schema                    |
+      | $.sink                  | /components/schemas/CreatePayment  |
+      | $.sinkCredential        | /components/schemas/SinkCredential |                  |
+
+
+  @create_payment_400.04_required_input_properties_missing
+  Scenario Outline: Required input properties are missing
+    Given the request body property "<input_property>" is not included
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+    Examples:
+      | input_property                                                    |
+      | $.amountTransaction                                               |
+      | $.amountTransaction.paymentAmount                                 |
+      | $.amountTransaction.referenceCode                                 |
+      | $.amountTransaction.paymentAmount.chargingInformation             |
+      | $.amountTransaction.paymentAmount.chargingInformation.amount      |
+      | $.amountTransaction.paymentAmount.chargingInformation.currency    |
+      | $.amountTransaction.paymentAmount.chargingInformation.description |
+
+
+  @create_payment_400.05_clientCorrelator_in_use
```

Comment:
Not related to Test Definition and to be discussed later but when I check the test I'm wondering if we have not some duplication between clientCorrelator & referenceCode (with test @create_payment_409.01_payment_duplicated) . Not sure why we have both of them. We can probably discuss this after the meta release.